Mascots: Update pyun's osu!taiko mascot entry and other stuff (#2188)
ppy/osu-wiki: wiki/Mascots/en.md

### ![osu!catch icon](/wiki/shared/mode/catch.png) Yuzu

> *For the news post, see: [Meet Yuzu](https://osu.ppy.sh/home/news/2014-06-21-meet-yuzu)*

Reviewer: Also add rework newspost and maybe update description with the changes.
```suggestion
*For the news post, see: [Meet Yuzu!](https://osu.ppy.sh/home/news/2014-06-21-meet-yuzu)*
```

Author: Alright, I'll add in the link to the "Introducing Yuzu's New Look" news post and add a note in the description that the current design was by Thievley. I'll try to rewrite the descriptions another time, unless someone else wants to do it or does it before I do.

> > Don is the protagonist of the [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin) series. He is a taiko drum with a white edge and four legs, a red face (which appears to be moly orange), and a light blue body. Don's dream is to share the beauty of Taiko with the world. Three years have passed since he moved into the Wada House, and he's become quite popular around town. He has a monstrous appetite and sometimes has expensive shopping at the Wada House which may take over for the worst. He usually ends his sentences with "Ta-don", meaning "Ba-dum" in Japanese.

Reviewer: If it was a quote taken from somewhere, leave as is.
`he's become` -> `he has become` (diff not showing)
```suggestion
> Don is the protagonist of the [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin) series. He is a taiko drum with a white edge and four legs, a red face (which appears to be moly orange), and a light blue body. Don's dream is to share the beauty of Taiko with the world. Three years have passed since he moved into the Wada House, and he has become quite popular around town. He has a monstrous appetite and sometimes has expensive shopping at the Wada House which may take over for the worst. He usually ends his sentences with "Ta-don", meaning "Ba-dum" in Japanese.
```

Author: It looks like this *might* have been mostly copied from [here](https://gamicus.gamepedia.com/Characters_of_Taiko_no_Tatsujin#WadaDon) ~~... or maybe they copied from us?~~ ¯\\\_(ツ)\_/¯

but since it looks like it was copied from there, I'll include some kind of link to that page.

和田どん (Wada Don), also known as Don or Don-chan, was one of the osu! mascots for osu!taiko that joined in 2008-05. He is 48 centimetres tall and weighs more than 100 kilograms. He appears in the skin for osu!taiko. His art was designed by Yukiko Yokoo (横尾有希子) and was voiced by Narahashi Miki (楢橋 美紀).

## Community

### Aiko

![Aiko](img/Aiko.png "Aiko")

> Designed by [JMC](https://osu.ppy.sh/users/774010), Aiko was one of the participants in the osu!taiko mascot design contest. She is an energetic girl with a passion for osu!taiko, even if she is not the best at it! Sporting a pair of stylish "Tabi" shoes and a retinue of pippidon accessories, the mascot of yore lives on in this spunky girl. She has fairly short at only 154 centimetres, and was born on 1999-04-06.

Reviewer: ```suggestion
Designed by [JMC](https://osu.ppy.sh/users/774010), Aiko was one of the participants in the osu!taiko mascot design contest. She is an energetic girl with a passion for osu!taiko, even if she is not the best at it! Sporting a pair of stylish "Tabi" shoes and a retinue of pippidon accessories, the mascot of yore lives on in this spunky girl. She is fairly short at only 154 centimetres, and was born on 1999-04-06.
```

### Alisa

![Alisa](img/Alisa.png "Alisa")

Designed by [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa was one of the participants in the osu!taiko mascot design contest. She has been playing osu!taiko ever since her early childhood. Fond of playing songs for others, she is very musically oriented, and when she is not sleeping or eating, she is playing osu!taiko, or playing retro video games for kicks!

### Chirou

![Chirou](img/Chirou.png "Chirou")

> Designed by [pyun](https://osu.ppy.sh/users/981534), Chirou was one of the participants in the osu!taiko mascot design contest. She is very strict and demanding, a perfectionist, and does not like making mistakes—especially in her beats in osu!taiko. But past her strong outlook, if you get to her soft spot, she can be fluffy and cute. She is 14 years old, born on October 25th with blood type AB. She is also 4 feet 11 inches and weighs 100 pounds. Her hobbies include practicing drumming, keeping warm in her cape, and collecting gems or rocks. Chirou has made some appearances in various user fanart submissions.

Reviewer: I think it should be metric (for consistency).
`4 feet 11 inches and weighs 100 pounds` -> `about 149.86 centimetres and weighs about 45.36 kilograms` (taken from previous commits)
```suggestion
Designed by [pyun](https://osu.ppy.sh/users/981534), Chirou was one of the participants in the osu!taiko mascot design contest. She is very strict and demanding, a perfectionist, and does not like making mistakes—especially in her beats in osu!taiko. But past her strong outlook, if you get to her soft spot, she can be fluffy and cute. She is 14 years old, born on October 25th with blood type AB. She is also about 149.86 centimetres and weighs about 45.36 kilograms. Her hobbies include practicing drumming, keeping warm in her cape, and collecting gems or rocks. Chirou has made some appearances in various user fanart submissions.
```

Reviewer 2: it's a character profile, typically these just include what the artist intended

not that units change the meaning but saying "149.86 cm" makes it look like that was intentional when it's not 🤔

Reviewer: probably round to whole number. otherwise, it's fine.

Designed by [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa was one of the participants in the osu!taiko mascot design contest. She has been playing osu!taiko ever since her early childhood. Fond of playing songs for others, she is very musically oriented, and when she's not sleeping or eating, she's playing osu!taiko, or playing retro video games for kicks!

### Chirou

![Chirou](img/Chirou.png "Chirou")

> Designed by [pyun](https://osu.ppy.sh/users/981534), Chirou was one of the participants in the osu!taiko mascot design contest. She is very strict and demanding, a perfectionist, and does not like making mistakes—especially in her beats in osu!taiko. But past her strong outlook, if you get to her soft spot, she can be fluffy and cute. She is 14 years old, born on October 25th with blood type AB. She is also 4 feet 11 inches (about 149.86 centimetres) and weighs 100 pounds (about 45.36 kilograms). Her hobbies include practicing drumming, keeping warm in her cape, and collecting gems or rocks. Chirou has made some appearances in various user fanart submissions.

Reviewer 3: what about leaving just centimeters and kilograms instead of all 4 metrics?
